tests/fuzzers/bls12381/bls12381_fuzz.go

```diff
@@ -217,7 +218,7 @@ func FuzzCrossG1MultiExp(data []byte) int {
 
 	// gnark multi exp
 	cp := new(gnark.G1Affine)
-	cp.MultiExp(gnarkPoints, gnarkScalars)
+	cp.MultiExp(gnarkPoints, gnarkScalars, ecc.MultiExpConfig{NbTasks: 1})
```

Comment:
Not important as this is for fuzzing, though note that if this is set to 0, then it will use all of the available CPUs: https://github.com/ConsenSys/gnark-crypto/blob/1a7a29904a7c29ba3e1c644d9752765b6ae9775e/ecc/bls12-381/multiexp.go#L78

Reply:
Yeah, I know. I set it to `0` since I suspect that the fuzz-framework already uses as many cores as are available.

Reply:
hm, actually, you're right. let's leave it at default
```suggestion
	cp.MultiExp(gnarkPoints, gnarkScalars, ecc.MultiExpConfig{})
```